src/pages/ListTableList/components/CreateForm.tsx

```diff
@@ -0,0 +1,40 @@
+import { Form, Input, Modal } from 'antd';
+
+import { FormComponentProps } from 'antd/es/form';
+import React from 'react';
+
+const FormItem = Form.Item;
+
+interface CreateFormProps extends FormComponentProps {
+  modalVisible: boolean;
+  onSubmit: (fieldsValue: { desc: string }) => void;
+  onCancel: () => void;
+}
+
+const CreateForm: React.FC<CreateFormProps> = props => {
+  const { modalVisible, form, onSubmit: handleAdd, onCancel } = props;
+  const okHandle = () => {
+    form.validateFields((err, fieldsValue) => {
+      if (err) return;
+      form.resetFields();
+      handleAdd(fieldsValue);
+    });
+  };
+  return (
+    <Modal
+      destroyOnClose
+      title="新建规则"
+      visible={modalVisible}
+      onOk={okHandle}
+      onCancel={() => onCancel()}
+    >
+      <FormItem labelCol={{ span: 5 }} wrapperCol={{ span: 15 }} label="描述">
+        {form.getFieldDecorator('desc', {
+          rules: [{ required: true, message: '请输入至少五个字符的规则描述！', min: 5 }],
+        })(<Input placeholder="请输入" />)}
+      </FormItem>
+    </Modal>
+  );
+};
+
+export default Form.create<CreateFormProps>()(CreateForm);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4665582)